pkg/connectorbuilder/connectorbuilder.go

```diff
@@ -77,6 +78,7 @@ type builder struct {
 	credentialManagers      map[string]CredentialManagerLimited
 	eventFeeds              map[string]EventFeed
 	accountManagers         map[string]AccountManagerLimited // NOTE(kans): currently unused
+	resourceActionManager   *actions.ResourceActionManager
```

Comment:
A recent strategy we have used for the connector builder is to apply a shim when we instantiate the struct so that their is a single way to do anything on the builder itself.  For instance, we don't have a _ResourceSyncer_ and a _ ResourceSyncerV2_.  - _addTargetedSyncer_ wraps the V1 version with _newResourceSyncerV1toV2_.  Otherwise, we have deeply branching logic within the gRPC handlers.  Is it possible to encapsulate the logic into a single interface on this side?  We could store the global actions under an empty key or the equivalent.

Reply:
@kans the `resourceActionManager` logic has been fully merged with the pre-existing `ActionManager` interface. I think this satisfies what you were asking for.